tests/json.rs

```diff
@@ -0,0 +1,370 @@
+use seccompiler::{apply_filter, compile_from_json, BpfProgram};
+use std::convert::TryInto;
+use std::env::consts::ARCH;
+use std::io::Read;
+use std::thread;
+
+const FAILURE_CODE: i32 = 1000;
+
+fn validate_json_filter<R: Read>(reader: R, validation_fn: fn(), should_fail: Option<bool>) {
```

Comment:
Can you update this as per: https://github.com/rust-vmm/seccompiler/pull/7#discussion_r687769473